src/core/util/getSlug.js

```diff
@@ -0,0 +1,15 @@
+import { slugify } from "transliteration";
```

Comment:
Remove this file. It's part of the `api-uitls` package

Reply:
Done